src/ui/page/fileListDownloaded/view.jsx

```diff
@@ -2,44 +2,42 @@
 import React from 'react';
 import Button from 'component/button';
 import FileList from 'component/fileList';
-import Page from 'component/page';
-import { PAGES } from 'lbry-redux';
 
 type Props = {
   fetching: boolean,
-  fileInfos: {},
-  sortBy: string,
+  downloadedUris: Array<string>,
 };
 
-class FileListDownloaded extends React.PureComponent<Props> {
-  render() {
-    const { fetching, fileInfos, sortBy } = this.props;
-    const hasDownloads = fileInfos && Object.values(fileInfos).length > 0;
+function FileListDownloaded(props: Props) {
+  const { fetching, downloadedUris } = props;
 
-    return (
-      // Removed the <Page> wapper to try combining this page with UserHistory
-      // This should eventually move into /components if we want to keep it this way
-      <React.Fragment>
-        {hasDownloads ? (
-          <FileList fileInfos={fileInfos} sortBy={sortBy} page={PAGES.DOWNLOADED} />
-        ) : (
-          <div className="main--empty">
-            <section className="card card--section">
-              <header className="card__header">
-                <h2 className="card__title">{__("You haven't downloaded anything from LBRY yet.")}</h2>
-              </header>
+  const hasDownloads = !!downloadedUris.length;
+  console.log('downloadedUris', downloadedUris);
```

Comment:
if I notice another one I won't comment since I know this will be linted eventually